dom0/sd-clean-all.sls

```diff
@@ -112,3 +112,12 @@ sd-cleanup-sys-firewall:
       - qvm-run sys-firewall 'sudo rm -f /rw/config/RPM-GPG-KEY-securedrop-workstation-test'
       - qvm-run sys-firewall 'sudo rm -f /etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation'
       - qvm-run sys-firewall 'sudo rm -f /etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation-test'
+
+disable-systemd-units:
+  cmd.run:
+    - name: systemctl --user disable sdw-notify.timer
```

Comment:
Noting for searchability/for our future selves: I think this is fine, but if we run into any errors we can try ` systemctl [...] || true` in `cmd.run` situations where we want to be sure we continue regardless, eg in cleanup. [saltstack-formula](https://github.com/saltstack-formulas/mongodb-formula/blob/fbe383183ff7742292e24f24ff734effe1a87384/mongodb/service/clean.sls#L24-L26) example